src/Controls/tests/UITests/Tests/Issues/Issue19831.cs

```diff
@@ -20,16 +21,29 @@ public void ActionModeMenuShouldNotBeVisibleAfterSwitchingTab()
 		this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
 
 		var rect = App.WaitForElement("Item1").GetRect();
-		var app = App as AppiumApp;
+	
+		PerformLongPress(rect);
```

Comment:
Can you move this into `AppiumTouchActions` and setup an extension method inside `AppiumApp` so this is reuseable?

Reply:
Let me do it.